.travis.yml

```diff
@@ -28,7 +27,7 @@ before_script:
   - ./travisci/bin/ci/setup.sh
   - composer install
 
-script: vendor/bin/phpunit --debug
+script: vendor/bin/phpunit --enforce-time-limit --debug
```

Comment:
It strictly requires the invoker package... this is something I won't add to the requirements after all. (It cost a lot of pain if your on win*** - don't ask why I know that...)